sjsonnet/test/src/sjsonnet/Std0150FunctionsTests.scala

```diff
@@ -100,6 +100,7 @@ object Std0150FunctionsTests extends TestSuite {
 
       check("""std.extVar("num")""", 1)
       check("""std.extVar("str")""", "hello")
+      check("""std.extVar("s" + "tr")""", "hello")
```

Comment:
Does this really test something?

Reply:
I think we should replace the `"s" + "tr"` with a function call to test what you want.